Vagrantfile

```diff
@@ -45,6 +46,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     }
   end
   mesos_zk_url = "zk://"+master_infos.map{|master| master[:ip]+":2181"}.join(",")+"/mesos"
+  marathon_zk_url = "zk://"+master_infos.map{|master| master[:ip]+":2181"}.join(",")
```

Comment:
that's handled inside the role as for the same ips it needs /marathon and /mesos